github-fetcher/json-orm.go

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
Can you check with someone with more golang experience about whether hypens or underscores are preferred for filenames? I would have expected underscores but it looks like I find both for golang files in this org.